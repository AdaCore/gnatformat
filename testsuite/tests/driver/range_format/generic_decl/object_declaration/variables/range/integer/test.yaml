description: ObjDecl formatting from line selection.
driver: gnatformat
args:
  [
    "test.ads",
    "--range-format",
    "-SL 64",
    "-EL 65",
    "--pipe",
  ]
