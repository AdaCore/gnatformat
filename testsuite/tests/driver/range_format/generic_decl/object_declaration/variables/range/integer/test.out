test.ads(<ObjectDecl ["Size"] test.ads:64:4-64:49>) - 64:1-64:49
^
   Size : Integer range 0 .. 10_000 := 0;$

